stackit/internal/services/sqlserverflex/user/resource_test.go

```diff
@@ -23,6 +23,9 @@ func TestMapFieldsCreate(t *testing.T) {
 				Item: &sqlserverflex.User{
 					Id:       utils.Ptr("uid"),
 					Password: utils.Ptr(""),
+					Roles: &[]string{
+						"role_1",
+					},
```

Comment:
In the "default_values" case this should be null, that's the default API response right?

Reply:
adapted the unit test